packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -65,10 +73,14 @@ const CreateANewOrganizationFormChild = ({ session }: { session: Ensure<SessionC
         // He won't need to have access to the org directly in this way.
         signIn("impersonation-auth", {
           username: data.email,
-          callbackUrl: `/settings/organizations/${data.organizationId}/about`,
+          callbackUrl: !isPlatformOrg
+            ? `/settings/organizations/${data.organizationId}/about`
+            : `/settings/platform`,
         });
       }
-      router.push(`/settings/organizations/${data.organizationId}/about`);
+      router.push(
+        !isPlatformOrg ? `/settings/organizations/${data.organizationId}/about` : "/settings/platform"
```

Comment:
`!isPlatformOrg` appears in 7 places in this file which to me indicates that platform creation form should be in a separate file - if its just 1 step it greatly differs from normal org IMO. What do you think?

Reply:
Thinking about it now, yes its actually a good idea!